lldb/tools/lldb-dap/Protocol/ProtocolTypes.h

```diff
@@ -269,17 +271,16 @@ struct Capabilities {
 };
 llvm::json::Value toJSON(const Capabilities &);
 
-enum PresentationHint : unsigned {
-  ePresentationHintNormal,
-  ePresentationHintEmphasize,
-  ePresentationHintDeemphasize,
-};
-llvm::json::Value toJSON(PresentationHint hint);
-
 /// A `Source` is a descriptor for source code. It is returned from the debug
 /// adapter as part of a `StackFrame` and it is used by clients when specifying
 /// breakpoints.
 struct Source {
+  enum PresentationHint : unsigned {
+    ePresentationHintNormal,
+    ePresentationHintEmphasize,
+    ePresentationHintDeemphasize,
+  };
```

Comment:
style nit: I think in lldb enums are usually kept in the same namespace and we should make the type more specific like `ScopePresentationHint` and `eScopePresentationHintNormal`, but I'll defer @JDevlieghere this.

Reply:
I'm slightly leaning towards that too, because that would be most consistent with the other enums which are potentially shared across types.